prediction/config.py

```diff
@@ -6,16 +6,23 @@
 """
 import os
 
+from os import path
+from glob import glob
+
+LIDC_WILDCARD = ['LIDC-IDRI-*', '**', '**']
+
 
 class Config(object):
     PROD_SERVER = os.getenv('PRODUCTION', False)
     DEBUG = False
-    # The following paths are expanded at runtime
-    CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
-    SEGMENT_ASSETS_DIR = os.path.abspath(os.path.join(CURRENT_DIR, 'src', 'algorithms', 'segment', 'assets'))
-    DICOM_PATHS_DOCKER_WILDCARD = os.path.join('/images_full', 'LIDC-IDRI-*', '**', '**')
-    DICOM_PATHS_LOCAL_WILDCARD = os.path.join(CURRENT_DIR, 'src', 'tests', 'assets',
-                                              'test_image_data', 'full', 'LIDC-IDRI-*', '**', '**')
+    CURRENT_DIR = path.dirname(path.realpath(__file__))
+    PARENT_DIR = path.dirname(CURRENT_DIR)
+    ALGOS_DIR = path.abspath(path.join(CURRENT_DIR, 'src', 'algorithms'))
+    SEGMENT_ASSETS_DIR = path.abspath(path.join(ALGOS_DIR, 'segment', 'assets'))
+    FULL_IMAGE_PATH = path.join(PARENT_DIR, 'images_full')
+    SMALL_IMAGE_PATH = path.join(PARENT_DIR, 'images')
+    FULL_DICOM_PATHS = glob(path.join(FULL_IMAGE_PATH, *LIDC_WILDCARD))
```

Comment:
This run's glob at import-time. Are we sure we want that?

Reply:
Good call. Moved the glob to `conftest.py`.